tasks/main.yml

```diff
@@ -44,7 +52,10 @@
     lc_ctype: "{{ postgres_locale }}"
     state: present
     template: template0
+  vars:
+    ansible_ssh_pipelining: true
   with_dict: "{{ postgres_users }}"
+  tags: postgres-create-db
```

Comment:
These tags are getting overly specific IMO. Since all tasks should be idempotent anything setting up Postgres can safely be grouped together and really should be. The time penalty is worth the “abstraction” here